xExtension-TitleWrap/static/title_wrap.css

```diff
@@ -0,0 +1,23 @@
+.horizontal-list {
+ display: flex;
+ /* uncomment to allow title to expand onto its own line */
+ /*flex-wrap: wrap;*/
+}
+.horizontal-list.bottom {
+ display: table;
+}
+.flux .item {
+ flex-shrink: 0;
+ line-height: normal;
+ padding: .3em 0;
+}
+.flux .item > a {
+ white-space: normal;
+}
+.flux:not(.current):hover .item.title {
+ position: relative;
+ max-width: inherit;
+}
+.flux_header .title {
+ flex: auto;
+}
```

Comment:
Just a note about indentation: I recommend to use 1 tabulation (i.e. FreshRSS coding style) or 4 spaces (my current preference). 1 space is a bit odd ^^